.github/workflows/Packages - EdFi.Ods.Minimal.Template.yml

```diff
@@ -0,0 +1,116 @@
+    - name: Checkout Ed-Fi-ODS
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Checkout Ed-Fi-ODS-Standard
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-Standard
+          path: Ed-Fi-Standard/
+          ref: ${{ env.EDFI_STANDARD_REFERENCE }}
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: powershell
+      run: |
+           $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+           $is_pull_request_branch = 'False'
+           $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+           $odsBranch = '${{ env.ODS_CURRENT_BRANCH }}'
+           if($odsBranch -ne ''){
+              git fetch origin $odsBranch
+              git checkout $odsBranch
+           } else {
+              if ($is_pull_request_branch -eq $true) {
+                git fetch origin ${{ env.CURRENT_BRANCH }}
+                git checkout ${{ env.CURRENT_BRANCH }}
+              }
+           }
+    - name: Install sql-server-2019 & sqlpackage
+      shell: powershell
+      run: |
+          choco install sql-server-2019  -y --params "'/IGNOREPENDINGREBOOT /IACCEPTSQLSERVERLICENSETERMS /Q /ACTION=install /INSTANCEID=MSSQLSERVER /INSTANCENAME=MSSQLSERVER /TCPENABLED=1 /UPDATEENABLED=FALSE /FEATURES=SQL,Tools'" --execution-timeout=$installTimeout
+          choco install sqlpackage
+    - name: Initialize-DevelopmentEnvironment
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      run: |
+        $ErrorActionPreference = 'Stop'
+        [Environment]::SetEnvironmentVariable('msbuild_buildConfiguration', 'Release')
+
+        $PSVersionTable
+          . $env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/Initialize-PowershellForDevelopment.ps1
+        
+        Initialize-DevelopmentEnvironment -NoCredentials -NoDeploy
+      shell: powershell
+    - name: Remove Plug-Ins
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      run: |
+        $ErrorActionPreference = 'Stop'
+        rm -R "./Plugin/EdFi.*"
+      shell: powershell 
+    - name: Create Database Template (no extensions)
+      working-directory: ./
+      run: |
+        $ErrorActionPreference = 'Stop'
+        [Environment]::SetEnvironmentVariable('msbuild_buildConfiguration', 'Release')
+        Import-Module -Force -Scope Global "$env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/DatabaseTemplate/Modules/create-minimal-template.psm1"      
```

Comment:
I think this may be worth visiting as a consolidation exercise in the future.

Reply:
Normally I would say we should make this change as part of this PR, since the Acceptance Criteria mention that we should move any code out of build steps and into powershell, but with how long this one has gone on unintentionally then I'm okay with it being handled separately.